body.tex

```diff
@@ -60,22 +60,22 @@ \section*{NumPy arrays}
 
 The \emph{shape} of an array determines the number of elements along each axis,
 and the number of axes is the array's dimensionality.
-For example, a vector of numbers can be stored as a one dimensional array of
-shape $N$, while color videos are four dimensional arrays of shape
+For example, a vector of numbers can be stored as a one-dimensional array of
+shape $N$, while color videos are four-dimensional arrays of shape
 $(T, M, N, 3)$.
 
 \emph{Strides} are necessary to interpret computer memory, which stores elements
 linearly, as multidimensional arrays.
 It describes the number of bytes to move forward in memory to jump from row to
-row, column to column, and so forth.
```

Comment:
I'll probably retain this for now, as the intent was to describe that this holds for >2 dimensions too.